doozer/doozerlib/backend/rebaser.py

```diff
@@ -465,7 +469,7 @@ class MergeSourceResult:
             # as /Dockerfile (which OSBS requires). Read in the content and write it back out
             # to the required distgit location.
             source_dockerfile_content = source_dockerfile.read()
-            distgit_dockerfile.write(source_dockerfile_content)
+            await exectools.to_thread(distgit_dockerfile.write, source_dockerfile_content)
```

Comment:
Thinking both `source_dockerfile.read()` and `distgit_dockerfile.write` can be made async.

Reply:
can be done with aiofiles or aioshutil